lib/compiler/passes/remove-proxy-rules.js

```diff
@@ -1,9 +1,10 @@
 "use strict";
 
+const asts = require("../asts");
 const visitor = require("../visitor");
 
 // Removes proxy rules -- that is, rules that only delegate to other rule.
-function removeProxyRules(ast, options) {
+function removeProxyRules(ast, options, session) {
```

Comment:
Actually, `session` seems to be more frequently used parameter, but adding it as the second would be a breaking change.